go.mod

```diff
@@ -102,7 +102,7 @@ require (
 	github.com/minio/highwayhash v1.0.2 // indirect
 	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/mtibben/percent v0.2.1 // indirect
-	github.com/onsi/gomega v1.20.0 // indirect
+	github.com/onsi/gomega v1.22.1 // indirect
```

Comment:
Why is this changed? Do you know?

Reply:
Because our proto script runs `go mod tidy`.